ui/lib/pki/addon/components/pki-issuer-cross-sign.hbs

```diff
@@ -0,0 +1,82 @@
+<h3 for="intermediates to cross-sign" class="title is-4">Intermediates</h3>
+<div class="box is-fullwidth has-only-top-shadow">
+  <p class="has-bottom-margin-xl">
+    Intermediates are identified by specifying the mount path and the issuer’s current name, as well as the new name it will
+    have when it is imported to Vault.
+  </p>
+  {{#if (gt this.signedIssuers.length 0)}}
```

Comment:
No need to change but you could simplify this to a truthy expression. The `if` in a template will treat an array with no length as falsy.
```hbs
{{#if this.signedIssuers}}
```

Reply:
Thank you! I think I mix this up with getters because I've had to explicitly check for `.length` there